FantasmoSDK/Classes/Extensions/Dictionary+Extension.swift

```diff
@@ -17,3 +17,11 @@ extension Dictionary {
         }
     }
 }
+
+extension Dictionary where Key == UUID {
```

Comment:
I see what this does, but what is it for?

Reply:
I have added comment with explanation - https://take.ms/VUSQf